lib/winston/logger.js

```diff
@@ -228,29 +231,28 @@ class Logger extends Transform {
       const tokens = msg && msg.match && msg.match(formatRegExp);
 
       if (!tokens) {
-        this.write(Object.assign({}, meta, {
-          [LEVEL]: level,
-          [SPLAT]: splat,
-          level,
-          message: msg
-        }, this.defaultMeta));
-      } else {
-        this.write(Object.assign({}, {
+        const info = Object.assign({}, meta, {
           [LEVEL]: level,
           [SPLAT]: splat,
           level,
           message: msg
-        }, this.defaultMeta));
+        }, this.defaultMeta);
```

Comment:
Shouldn't order be the other way around? This looks like default meta will override provided one.

Reply:
That's a good catch, but this is unrelated to this PR. It was introduced in https://github.com/winstonjs/winston/pull/1471

Reply:
My mistake – it was introduced in https://github.com/winstonjs/winston/pull/1552.